tests/unit/src/test/scala/org/scalafmt/tests/CoverageSuite.scala

```diff
@@ -0,0 +1,253 @@
+
+  // meta.Term
+  check("this")                       // Term.This
+  check("a.this")                     // Term.This
+  check("a.super.b")                  // Term.Super
+  check("super[a].b")                 // Term.Super
+  check("a.super[b].c")               // Term.Super
+  check("a")                          // Term.Name
+  check("a.b")                        // Term.Select
+  check("-a")                         // Term.ApplyUnary
+  check("+a")                         // Term.ApplyUnary
+  check("~a")                         // Term.ApplyUnary
+  check("!a")                         // Term.ApplyUnary
+  check("a(b)")                       // Term.Apply
+  check("a { case x => x }")          // Term.Apply
+  check("a { b }")                    // Term.Apply
+  check("a[B]")                       // Term.ApplyType
+  check("a + a")                      // Term.ApplyInfix
+  checkStructural("a + (())")         // Term.ApplyInfix
+  check("(a, b) + c")                 // Term.ApplyInfix
+  check("a + (b, c)")                 // Term.ApplyInfix
+  check("a + ((b, c))")               // Term.ApplyInfix
+  check("a = 1")                      // Term.Assign
+  check("return b")                   // Term.Return
+  check("throw e")                    // Term.Throw
+  check("a: A")                       // Term.Ascribe
+  check("x: @u")                      // Term.Annotate
+  check("(1, 1)")                     // Term.Tuple
+  checkStructural("{ a; b }")         // Term.Block
+  check("if (p) t ")                  // Term.If
+  check("if (p) {}")                  // Term.If
+  check("if (p) if (p2) t")           // Term.If
+  check("if (p) t else f")            // Term.If
+  check("x match { case _ => }")      // Term.Match
+  check("try f finally {}")           // Term.Try
+  check("try f catch { case _ => }")  // Term.Try
+  check("try f catch h")              // Term.TryWithHandler
+  check("try f catch h finally {}")   // Term.TryWithHandler
+  check("(a, b) => a + b")            // Term.Function
+  check("{ case _ => }")              // Term.PartialFunction
+  check("while (p) { d }")            // Term.While
+  check("do { d } while (p)")         // Term.Do
+  check("for { x <- xs } f(x)")       // Term.For
+  check("for { x <- xs } yield f(x)") // Term.ForYield
+  check("(new A)")                    // Term.New
+  check("new A(1)")                   // Term.New
+  check("new A {}")                   // Term.NewAnonymous
+  check("_")                          // Term.Placeholder
+  check("f _")                        // Term.Eta
+  check("f(x: _*)")                   // Term.Repeated
+  check("def f(x: A): B")             // Term.Param
+  check("def f(x: A = 1): B")         // Term.Param (default parameter)
+  check("s\"a $b\"")                  // Term.Interpolate
+  check("<a>b {c}</a>")               // Term.Xml
+
+  // meta.Pkg
+  checkSource("package a")
+  checkSourceStructural(
+    """|package a.b {
+     |  package c {
```

Comment:
Because `// format: off`, scalafmt won't align this `|` with `"""|`. I would prefer `align = more` if that avoids commenting on formatting.